taskotron_python_versions/two_three.py

```diff
@@ -97,7 +97,7 @@ def check_two_three(package):
                     log.debug('Requires Python {}'.format(py_version))
                     py_versions[py_version] = name
 
-    package.set_python_versions(py_versions)
+    package.py_versions = py_versions.keys()
```

Comment:
Please make it a set, so the underlying dict may die. (In Python 3 at least) the `.keys()` produces just a view that is bound to the dictionary. In Python 2 this produces a list, so by casting ti to set we at least know that we have the same data structure in both Pythons.